test/jdk/com/sun/net/httpserver/Test9.java

```diff
@@ -178,20 +188,18 @@ public boolean verify (String s, SSLSession s1) {
                 is.close();
                 fout.close();
 
-                if (count != size) {
+                if (count != filePath.toFile().length()) {
                     System.out.println ("wrong amount of data returned");
                     System.out.println ("fixedLen = "+fixedLen);
                     System.out.println ("protocol = "+protocol);
-                    System.out.println ("root = "+root);
                     System.out.println ("port = "+port);
-                    System.out.println ("f = "+f);
-                    System.out.println ("size = "+size);
+                    System.out.println ("file = " + filePath);
                     System.out.println ("temp = "+temp);
                     System.out.println ("count = "+count);
                     error = true;
                 }
-                String orig = root + "/" + f;
-                compare (new File(orig), temp);
+                Path tempPath = temp.toPath();
+                assert Files.mismatch(filePath, tempPath) < 0;
```

Comment:
And in all other places where this pattern appears in this PR

Reply:
Fixed in 9c7c6af013e0495b08bcd248e30d83385bcc35b5.